buildSrc/src/main/kotlin/IosTargetsExt.kt

```
@@ -0,0 +1,61 @@
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import org.gradle.api.Project
+import java.io.File
+import java.util.*
+
+// can't use by lazy, we need Project.findProperty not accessible there
+@Suppress("ObjectPropertyName")
+private lateinit var _iosTargets: List<String>
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)